### PR TITLE
Add badges

### PR DESCRIPTION
**Pull request by ahuang11**
https://github.com/PrefectHQ/prefect-collection-template/issues/13

see https://github.com/PrefectHQ/prefect-aws/pull/72 for example images

### CODE REVIEW

**Comment by ahuang11** [comment]
Tested generation successfully:
```
[![PyPI version](https://badge.fury.io/py/prefect-aws.svg)](https://pypi.python.org/pypi/prefect-aws/)
[![PyPI pyversions](https://img.shields.io/pypi/pyversions/prefect-aws.svg)](https://pypi.python.org/pypi/prefect-aws/)
[![Downloads](https://pepy.tech/badge/prefect-aws)](https://pepy.tech/project/prefect-aws)
[![Stars](https://img.shields.io/github/stars/prefecthq/prefect-aws)](https://github.com/prefecthq/prefect-aws/)
[![PyPI license](https://img.shields.io/pypi/l/prefect-aws.svg)](https://pypi.python.org/pypi/prefect-aws/)
```

image

**Comment by ahuang11** [comment]
image

**Comment by ahuang11** [comment]
Should we add discourse and slack?
https://img.shields.io/discourse/users?server=https%3A%2F%2Fdiscourse.prefect.io
```
    <a href="https://discourse.prefect.io/"" alt="Discourse">
        <img src="https://img.shields.io/discourse/users?server=https%3A%2F%2Fdiscourse.prefect.io" /></a>
```

**Comment by desertaxle** [comment]
Yeah, I think adding Discourse and Slack would be great

**Comment by ahuang11** [comment]
image

File: {{cookiecutter.collection_name}}/README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # {{ cookiecutter.collection_name }}
 
+<a href="https://github.com/{{ cookiecutter.github_organization }}/{{ cookiecutter.collection_name }}/" alt="Stars">
+    <img src="https://img.shields.io/github/stars/{{ cookiecutter.github_organization }}/{{ cookiecutter.collection_name }}" /></a>
```
Comment (desertaxle) [inline {{cookiecutter.collection_name}}/README.md:4]: I think this needs to be replaced with a PyPI version badge

**Review by desertaxle — CHANGES_REQUESTED** [review]

**Comment by ahuang11** [comment]
image

**Review by desertaxle — APPROVED** [review]

**Pull request merged**